packages/xo-web/src/xo-app/new-vm/index.js

```diff
@@ -496,9 +496,15 @@ export default class NewVm extends BaseComponent {
       })
     })
     if (VIFs.length === 0) {
-      VIFs.push({
-        network: this._getDefaultNetworkId(template),
-      })
+      const automaticNetworks = this._getAutomaticNetworks().map(network => ({
+        network: network.id,
+      }))
+
+      automaticNetworks.length !== 0
+        ? VIFs.push(...automaticNetworks)
+        : VIFs.push({
+            network: this._getDefaultNetworkId(template),
+          })
```

Comment:
This could be:
```js
VIFs = automaticNetworks.length !== 0
  ? automaticNetworks
  : [{ network: this._getDefaultNetworkId(template) }]
```
for better perf, I guess.